content/applications/hr/fleet/configuration.rst

```diff
@@ -0,0 +1,234 @@
+- :guilabel:`Model Year`: Enter the year the vehicle was manufactured.
+- :guilabel:`Trailer Hitch`: Check this box if the vehicle has a trailer hitch.
+
+Salary
+******
+
+- :guilabel:`Can be requested`: Check this box if employees can request this model vehicle.
+- :guilabel:`Catalog Value (VAT Incl.)`: Enter the :abbr:`MSRP (Manufacturer's Suggested Retail
+  Price)` for the vehicle at the time of purchase or lease.
+- :guilabel:`C02 fee`: This value is specific to Belgian based companies only. If logged into a
+  Belgian company, this value is automatically calculated based on Belgian laws and regulations, and
+  cannot be modified. The value is based on the value entered in the :guilabel:`CO2 Emissions` field
+  in the :guilabel:`Engine` section of the vehicle form. Modifying the :guilabel:`CO2 Emissions`
+  field will adjust the value in the :guilabel:`CO2 fee` field.
+- :guilabel:`Cost (Depreciated)`: Enter the monthly cost for the vehicle, which appears in the
+  salary configurator. This value impacts the gross and net salary of the employee who is assigned
+  to the vehicle. This figure is depreciated over time.
+- :guilabel:`Total Cost (Depreciated))`: This value is the :guilabel:`Cost (Depreciated)` and the
+  :guilabel:`C02 fee` fields combined, and also is depreciated over time.
+
+.. note::
+   The :guilabel:`Cost (Depreciated)` does not depreciate automatically on the vehicle model, it
+   only depreciates based on the contract linked to a specific vehicle and not on the general model.
+
+Engine
+******
+
+- :guilabel:`Fuel Type`: Select the type of fuel the vehicle uses form the drop-down menu. The
+  options are :guilabel:`Diesel`, :guilabel:`Gasoline`, :guilabel:`Hybrid Diesel`, :guilabel:`Hybrid
+  Gasoline`, :guilabel:`Plug-in Hybrid Diesel`, :guilabel:`Plug-in Hybrid Gasoline`,
+  :guilabel:`CNG`, :guilabel:`LPG`, :guilabel:`Hydrogen`, or :guilabel:`Electric`.
+- :guilabel:`CO2 Emissions`: Enter the carbon dioxide emissions the vehicle produces in grams per
+  kilometer (g/km).
+- :guilabel:`CO2 Standard`: Enter the standard amount of carbon dioxide for a similar sized vehicle.
+- :guilabel:`Transmission`: Select the transmission type from the drop-down menu, either
+  :guilabel:`Manual` or :guilabel:`Automatic`.
+- :guilabel:`Power`: If the vehicle is electric or hybrid, enter the power the vehicle uses in
+  kilowatts (kW).
+- :guilabel:`Horsepower`: Enter the vehicle's horsepower in this field.
+- :guilabel:`Horsepower Taxation`: Enter the amount that is taxed based on the size of the vehicles
+  engine. This is determined by local taxes and regulations, and varies depending on the location.
+  It is recommended to check with the accounting department to ensure this value is correct.
+- :guilabel:`Tax Deduction`: This field auto-populates according to the engine specifications, and
+  cannot be modified. The percentage is based on the localization settings and local tax laws.
```

Comment:
This is a repeat of the other doc. need to differentiate the two I am confused--- which populates which, one can be removed?

Reply:
These are actually different forms- one is when adding a new vehicle, this is when adding a new model in general. It is really confusing because 80% of the form is identical- BUT the form name is actually different, so it is not the same. It just looks a lot like the other!